backend/src/main/java/com/devdive/backend/post/adapter/in/web/PostController.java

```diff
@@ -0,0 +1,28 @@
+package com.devdive.backend.post.adapter.in.web;
+
+import com.devdive.backend.post.application.dto.PostCreateRequestDto;
+import com.devdive.backend.post.application.port.in.PostUseCase;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequiredArgsConstructor
+public class PostController {
+
+    private final PostUseCase useCase;
+
+    @PostMapping("/v1/posts")
+    ResponseEntity<Void> createPost(@RequestBody @Valid PostCreateRequestDto dto) {
+        useCase.createPost(dto);
+        return new ResponseEntity<>(HttpStatus.OK);
+    }
+
+    @GetMapping("/v1/posts/{postId}")
```

Comment:
/api가 빠졌고 클래스에 `@RequestMapping("/api/v1/posts")`를 추가해서 포스트 관련 요청을 처리하도록 바꾸는게 좋을듯 합니다.
```suggestion
@RestController
@RequestMapping("/api/v1/posts")
@RequiredArgsConstructor
public class PostController {

    private final PostUseCase useCase;

    @PostMapping
    ResponseEntity<Void> createPost(@RequestBody @Valid PostCreateRequestDto dto) {
        useCase.createPost(dto);
        return new ResponseEntity<>(HttpStatus.OK);
    }

    @GetMapping("/{postId}")
```